mod-search/src/test/resources/spitfire/mod-search/set-up/add-okapi-permissions.feature

```diff
@@ -0,0 +1,74 @@
+Feature: prepare data for api test
```

Comment:
Why do we need this?

Reply:
Without these permissions it was not possible to run tests locally in the development environment. Perhaps they are only needed for local running.